spec/support/stub_fast_connection.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+RSpec.shared_context('with FAST connection') do
```

Comment:
Put the FAST testing machinery in a shared context so it can be used by the FAST request spec and feature specs.